### ISSUE TITLE
Update ordering on task saved filters

### ISSUE THREAD

**Issue by NoopDog** [clevercanary/hca-atlas-tracker#333]
### Need
To help present the tasks on a coherent plan, we can order the tasks by additional fields besides the target completion date ascending on the task saved filters.

### Approach

For all saved filters

![Image](https://github.com/clevercanary/hca-atlas-tracker/assets/3239697/7235b350-0db5-4eab-9f25-a69b41166770)

We currently order by Target Completion ascending. Instead order by:

1. Target Completion ascending then by
2. Atlas ascending
3. Then by status

Once this is in place we can do a separate ticket to custom order the status or do the fake ordering/mapping we did for completed at.
